src/live_portrait_pipeline.py

```diff
@@ -187,9 +187,10 @@ def execute(self, args: ArgumentConfig):
 
         # save drived result
         wfp = osp.join(args.output_dir, f'{basename(args.source_image)}--{basename(args.driving_info)}.mp4')
+        wfp_sound = osp.join(args.output_dir, f'{basename(args.source_image)}--{basename(args.driving_info)}--sound.mp4')
```

Comment:
Thanks for your PR.  We also plan to add a flag_add_sound option as the argument, and will consider your implementation💖